packages/cli/solc/solc

```diff
@@ -14,6 +14,7 @@ var path = require('path');
 var wrapper = require('solc-wrapper');
 var binary = require('./soljson-v0.5.9+commit.e560f70d');
 var solc = wrapper(binary);
+var solcCompile = solc.compile ? solc.compile : solc.compileStandardWrapper;
```

Comment:
Is this necessary? From reading the source it seems like the name is always `compile`.

https://github.com/spalladino/solc-js/blob/e1645d9140133e3f7ba42b29b91c9863103e9b21/wrapper.js#L326

Reply:
Probably not, but wanted to be covered.